apps/frontend/src/api/hooks/useSms.ts

```diff
@@ -0,0 +1,12 @@
+import { ISms } from '@innbyggerpanelet/api-interfaces';
+import { poster } from '../operations';
+
+export const sendSms = async (sms: ISms) => {
+    const { data, error } = await poster<ISms>('/api/sms', sms);
+
+    return {
+        response: data,
+        isLoading: !error && !data,
+        isError: error
+    };
+};
```

Comment:
As mentioned in SmsRoutes and discussed in the office. I believe we should prevent SMS from being triggered through the frontend, and rather keep it backend centric.